src/platforms/web/server.js

```diff
@@ -63,7 +63,9 @@ const defaultHead =
   '<meta property="og:title" content="LBRY On The Web" />\n' +
   '<meta property="og:site_name" content="LBRY.tv"/>\n' +
   '<meta property="og:description" content="All your favorite LBRY content in your browser." />\n' +
-  '<meta property="og:image" content="/og.png" />';
+  '<meta property="og:image" content="https://beta.lbry.tv/og.png" />';
```

Comment:
What doesn't work about this?

Reply:
so what is the correct string i must write on 66 line Mr Thomas

Reply:
it already has og.png....

Reply:
"I think this should be: <meta property="og:image" content="https://beta.lbry.tv/og.png" /> " julie said on issues it should like as she said..

Reply:
This just means it's not being set on the main https://beta.lbry.tv URL, not because it's missing the full path. I'll file that as a separate issue.

Reply:
https://github.com/lbryio/lbry-desktop/pull/2953/commits/7d6d117b95e881d36e12876361b3884dd4fe4765 @tzarebczan i replace with default og image on repository folder.